lang/java.md

```diff
@@ -1,25 +1,27 @@
 ---
-    lang: Java
-    syntax: java
-    title: Java is Best
-    author: KhafiaAyyub
+lang: Java
+syntax: java
+title: Java is a piping hot cup of coffee!
+author: KhafiaAyyub
 ---
 
 ```java
-    public class HelloWorld {
-        public static void main(String[] args) {
-            System.out.println("Hello, World!"); 
-        }
+public class HelloWorld {
+    public static void main(String[] args) {
+        System.out.println("Hello, World!"); 
     }
+}
 ```
 
-## What is Java?
+## What is Java❓
```

Comment:
This question mark doesn't look good 😅 like , it just looks bad 🤷‍♂️ using emoji anywhere would be better, but not to replace other English characters...

Reply:
@vsnthdev: Any other emoji to recommend for this & below two?